client/finality-grandpa/src/justification.rs

```diff
@@ -42,9 +43,25 @@ use crate::{AuthorityList, Commit, Error};
 /// nodes, and are used by syncing nodes to prove authority set handoffs.
 #[derive(Clone, Encode, Decode, PartialEq, Eq, Debug)]
 pub struct GrandpaJustification<Block: BlockT> {
```

Comment:
```suggestion
pub struct GrandpaJustification<Header: HeaderT> {
```

Reply:
I also want to use it in this way. But its impls must use Block type.